python/scripts/process_osm.py

```diff
@@ -48,18 +49,24 @@ def run_osmium_cmd(flag: str, input_file: str, output_file: str = None):
                     logger.info("Renumbering of OSM data completed.")
             os.remove(tmp_file)
 
-def run_osm2pgsql_cmd(config: CredentialsConfig, input_file: str, style_file_path: str, coords: str| list[int] = None):
-    """Import data from input_file using osm2pgsql."""
+def run_osm2pgsql_cmd(config: CredentialsConfig, input_file: str, style_file_path: str, schema: str, force: bool, coords: str| list[int] = None):
+    """Import data from input_file to database specified in config using osm2pgsql tool."""
 
     if hasattr(config, "server"): # remote connection
-        ssh_tunnel_port = 1113 # check roadgraphtool.db.ssh_tunnel_local_port (needs to be same)
+        ssh_tunnel_port = db.ssh_tunnel_local_port
         db.set_ssh_to_db_server_and_set_port()
     else:  # local connection
         ssh_tunnel_port = config.db_server_port
 
-    cmd = ["osm2pgsql", "-d", config.db_name, "-U", config.username, "-W", "-H", config.db_host, 
-            "-P", str(ssh_tunnel_port), "--output=flex", "-S", style_file_path, input_file, "-x"]
+    if not force and not check_empty_or_nonexistent_tables(schema, config):
+        raise TableNotEmptyError("Attempt to overwrite non-empty tables. Use '--force' flag to proceed.")
+
+    if not schema_exists(schema, config):
```

Comment:
you can replace this whole logic with SQL query "CREATE SCHEMA IF NOT EXIST ...". Is there a reason you did not use this?

Reply:
you're right, I just didn't realize it at the time. Thanks for pointing it out, I'll fix that.